website/docs/r/ecs_service.html.markdown

```diff
@@ -136,6 +154,14 @@ The following arguments are optional:
 * `triggers` - (Optional) Map of arbitrary keys and values that, when changed, will trigger an in-place update (redeployment). Useful with `timestamp()`. See example above.
 * `wait_for_steady_state` - (Optional) If `true`, Terraform will wait for the service to reach a steady state (like [`aws ecs wait services-stable`](https://docs.aws.amazon.com/cli/latest/reference/ecs/wait/services-stable.html)) before continuing. Default `false`.
 
+### alarms
+
+The `alarms` configuration block supports the following:
+
+* `alarms_names` - (Required) One or more CloudWatch alarm names. Use a "," to separate the alarms.
```

Comment:
Since you're using terraform array/list to pass alarm names, the comment about using a comma seems unnecessary and suggests that would rather be a comma-separated string?
```suggestion
* `alarms_names` - (Required) One or more CloudWatch alarm names.
```

Reply:
Good point.